src/main/java/com/mael/payroll/paymentTypes/HourlyPayment.java

```diff
@@ -38,7 +38,7 @@ public TimeCard getTimeCard(LocalDate date) {
     public double calculatePay(LocalDate payDay) {
         double pay = 0.0;
         for(TimeCard timeCard : timeCards) {
-            if (inCurrenWeek(payDay, timeCard)) {
+            if (isInPeriod(payDay, timeCard)) {
```

Comment:
I liked `inCurrentWeek()` more because it was clear to me when reading it, that it has to be in the current week. Now I have to follow the method body of it in order to know what a _period_ means in this context.